pyproject.toml

```diff
@@ -17,11 +17,15 @@ repository = "https://github.com/sb-ai-lab/Sim4Rec"
 python = ">=3.8, <3.10"
 pyarrow = "*"
 sdv = "0.15.0"
-torch = "*"
```

Comment:
why do you introduce changes here?

Reply:
pls remove if those changes are not necessary, we want the framework to work with a wide range of dependancies' versions

Reply:
Ok, I'll check if this works. I don't remember exactly why I changed these versions, but I'm pretty sure that there are some incompatibilities without it.

Reply:
I removed all dependencies except torch/torchvision, looks like everything works fine. I can not remove  torch=="1.9.1" and "torchvision==0.10.0", because it is the last version that supports cuda 10.2. I work in docker container and I can't update cuda on the host machine, because this will ruin my colleague's workflow. On the other hand, nvidia-docker requires an exact match between cuda cersions inside container and on the host machine, otherwise it doesn't see my GPU.  

Although I can't test it, torch seems to be backward compatible and it **might** work well for newer versions if it works for 1.9.1; I will remove the dependencies

Reply:
done